app/api/schema/event_statistics.py

```diff
@@ -76,9 +76,19 @@ def sessions_canceled_count(self, obj):
         ).count()
 
     def speakers_count_type(self, obj, state='pending'):
-        return SessionsSpeakersLink.query.filter_by(
-            event_id=obj.id, session_state=state, deleted_at=None
-        ).count()
+        return (
+            Speaker.query.join(
+                SessionsSpeakersLink, SessionsSpeakersLink.speaker_id == Speaker.id
+            )
+            .join(Session, SessionsSpeakersLink.session_id == Session.id)
```

Comment:
```suggestion
            Speaker.query.join(Speaker.sessions)
```

Reply:
Hey @iamareebjamal  I tried with this but it's not fetching it correctly then

Reply:
Strange. This query works for me

```python
Speaker.query.join(Speaker.sessions).distinct(Speaker.id).filter(Speaker.deleted_at == None, Session.state=='confirmed').count()
```

Reply:
@iamareebjamal ohh my bad had made a mistake now :sweat_smile:  have updated mr :v: Thanks for the suggestion!